frontend/src/views/applications/ApplicationConfirmationView.vue

```diff
@@ -1,56 +1,100 @@
 <template>
   <v-container>
-    <h2 class="mb-8" align="center">OFM Application Submitted</h2>
-    <v-card elevation="2" class="supp-applications-card">
-      <v-card-text class="px-8" py-6>
-        <v-row no-gutters align="center">
-          <v-col cols="12" md="8">
-            <p>You may be eligible for supplementary funding. Would you like to apply?</p>
-            <p class="mt-4">Available allowances:</p>
-            <ul class="pl-6">
-              <li>Indigenous programming</li>
-              <li>Support needs programming</li>
-              <li>Transportation allowance</li>
-            </ul>
-            <p class="mt-4">
-              You may apply for one or more allowances. You can also apply through
-              <router-link :to="{ name: 'applications-history' }">Applications</router-link>
-              at any time.
-            </p>
-          </v-col>
-          <v-col cols="12" md="4" align="right" class="my-4">
-            <AppButton id="apply-button" :primary="false" size="large" width="100px" :to="{ name: 'supp-allowances-form', params: { applicationGuid: $route.params.applicationGuid } }">
-              Apply
-            </AppButton>
-          </v-col>
-        </v-row>
-      </v-card-text>
-    </v-card>
-    <v-row justify="space-around" no-gutters class="mt-10">
-      <AppBackButton id="back-home-button" :primary="false" width="250px" :to="{ name: 'home' }" align="center">Home</AppBackButton>
-      <AppButton id="view-applications-button" size="large" width="250px" :to="{ name: 'applications-history' }" align="center" class="mt-2">View Applications</AppButton>
-    </v-row>
+    <h2 class="mb-8 text-center">OFM Application Submitted</h2>
+    <v-skeleton-loader v-if="loading" :loading="loading" type="table-tbody"></v-skeleton-loader>
+    <div v-else>
+      <v-row class="flex justify-end">
+        <v-col cols="12">
+          <v-alert v-if="!hasGoodStanding" type="info" dense text>To apply, you must be in good standing with BC Registries.</v-alert>
+        </v-col>
+      </v-row>
+
+      <v-card elevation="2" class="supp-applications-card">
+        <v-card-text class="px-8" py-6>
+          <v-row no-gutters class="text-center">
+            <v-col cols="12" md="8">
+              <p>You may be eligible for supplementary funding. Would you like to apply?</p>
+              <p class="mt-4">Available allowances:</p>
+              <ul class="pl-6">
```

Comment:
can we improve its alignment?
![image](https://github.com/bcgov/ecc-ofm/assets/119005543/65c18334-1333-43a4-9490-729f32b80233)

Reply:
fixed :)

Reply:
looks great now!